tests/src/autoload/functions.php

```diff
@@ -777,14 +777,20 @@ function loadDataset()
  * @param string  $type
  * @param string  $name
  * @param boolean $onlyid
+ * @param boolean $throw_exception Whether to throw an exception if item is not found
  * @return CommonDBTM|false the item, or its id
  */
-function getItemByTypeName($type, $name, $onlyid = false)
+function getItemByTypeName($type, $name, $onlyid = false, bool $throw_exception = false)
```

Comment:
See #18952, an exception will be thrown now any time an item is not found. Your change is no longuer required.